lib/linguist/languages.yml

```
@@ -5512,6 +5512,15 @@ RAML:
   extensions:
   - ".raml"
   language_id: 308
+RBS:
+  type: programming
```

This should really be categorised as data, considering [RBS syntax](https://github.com/ruby/rbs/blob/b7923d37ef8efa514aaf1624763b1408e3450bd5/docs/syntax.md#syntax) is purely descriptive in nature:

```suggestion
  type: data
```